pages/docs/concepts/server.md

```diff
@@ -0,0 +1,15 @@
+
+## What is a server?
+A server is a representation of a messaging broker system where connections and communication between a producer and a consumer is established. Unlike traditional API servers which are depended on request/response, message brokers interactions occur back and forth over different TCP/IP.
+
+## What is the purpose of servers?
+Servers play an important role as they maintain a relationship between producers and consumers. When designing and setting up an event-driven application, servers are in charge of delivering asynchronous messages from the producer to the consumer through the use of channels. With the integration of different messaging protocols, servers can transmit and exchange messages between server-client connections or server to server connections.
+
+A. server diagram made by figma :
+
+<img width="1504" alt="Server_Diagram" src="https://user-images.githubusercontent.com/66913810/174148274-1299dc70-ef12-4f30-b3ed-01c260a8f697.png">
+
+B. server diagram made by mermaid.js :
+
+![image](https://user-images.githubusercontent.com/66913810/176436057-c045b082-7381-4bf1-9c28-96ee3df4707a.png)
```

Comment:
Yes @thulieblack, I know [we're still voting as a community on what diagraming tool we'll use](https://github.com/asyncapi/website/issues/849)e, but if you're already 100% comfortable and have fully copied your original FigJam diagram over to MermaidJS syntax... then it's def better to include the actual markdown of your mermaidJS diagram here in your PR so that reviewers can make commit suggestions with greater ease. 😀